cosmos/providers/dbt/render.py

```diff
@@ -74,7 +77,9 @@ def render_project(
             )
 
     # iterate over each model once to create the initial tasks
-    for model_name, model in project.models.items():
+    for model_name, model in itertools.chain(
+        project.models.items(), project.snapshots.items(), project.seeds.items()
+    ):
```

Comment:
Hi

I wanted to link this to issue #235, which I opened a few days ago. It seems that data is not being seeded before the DbtTaskGroup collects the DBT models. Even after updating to release 0.6.0 (which includes this commit), the problem is still present.

After reviewing the code, I'm wondering if we should consider reordering this chain so that seeding tasks are created first, as shown below:

```
for model_name, model in itertools.chain(
    project.snapshots.items(), 
    project.seeds.items(),
    project.models.items(), 
):
```

I would appreciate your thoughts on this potential solution, and any other insights you may have to address this issue.

Thank you!

Reply:
Hi @DucNgn,

Unfortunatelly I couldn't reproduce the issue #235 in release 0.6.0.
It build dependency and task order from DBT metadata and not from the running order of the for loop. 

I tested with slightly modificated jaffle demo project, which has got the following structure:
├── models
│   ├── customers.sql
│   ├── docs.md
│   ├── orders.sql
│   ├── overview.md
│   ├── schema.yml
│   ├── sources.yml
│   └── staging
│       ├── schema.yml
│       ├── stg_customers.sql
│       ├── stg_orders.sql
│       └── stg_payments.sql
├── seeds
│   ├── raw_customers.csv
│   ├── raw_orders.csv
│   └── raw_payments.csv
├── snapshots
   └── orders.sql

And it generated the following Airflow task group:
![image](https://user-images.githubusercontent.com/18029373/232615984-33e6feff-b44c-4788-b83d-a2f4b922c398.png)

I used the **ref** keyword in **stg_customers.sql** to reference the **raw_customers.csv** seed:
`select * from {{ ref('raw_customers') }}`
How do you reference to your seed?

If you see "Dependency abc not found for model DbtModel..." message then probably entities Dict doesn't contain the neccessary dependency: https://github.com/astronomer/astronomer-cosmos/blob/astronomer-cosmos-v0.6.0/cosmos/providers/dbt/render.py#L209
It probably means you failed to read your seeds files. If these .csv files doesn't located in your DBT project snapshots subfolder then you have to specify with dbt_snapshots_dir param in DbtTaskGroup. For example: 
```
    dbt_tg = DbtTaskGroup(
        group_id="dbt_tg",
        dbt_project_name="jaffle_shop",
        dbt_root_path="your_root_path_location",
        dbt_snapshots_dir="your_snapshot_dir_location",
        conn_id="your_pg_connection",
        dbt_args={
            "schema": "public",
        },
    )
```
Where are your DBT seed files located? 

I hope I could help you. Please provide more details about the structure of the project and dependencies in case the error still persists.

Reply:
Hi @szecsip,

I wanted to thank you for your detailed explanation in response to my issue with `DbtTaskGroup`. After reviewing the code again, I believe the problem is that Cosmos is not currently reading `dbt_project.yml` to determine the path to the seed files. This is likely the reason for the errors I'm experiencing.

While we don't currently have a way for clients to change the path to a different seeding folder, I'm planning to open a PR that adds an argument to specify the location of the seed files.

Reply:
Hi @DucNgn ,

I forgot to take out seed_dir argument in few places, but I have already prepared it. If you don't mind I will send a PR to #235 in a day.

Reply:
Sorry, I didn't see this message earlier. I opened #241 to tackle this already. 
It's just a small change, so no big deals.

Reply:
No problem, thank you for the supplement and congrats on the PR.